app/views/cart_tasks/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <h1>Welcome to your Cart</h1>
   <div class="checkout-button">
     <% if current_volunteer %>
-    <%= button_to "Checkout", cart_path, method: :get, class:"slider-btn waves-effect waves-light btn"%>
+    <%= button_to "Checkout", commitments_path(), method: :post, class:"slider-btn waves-effect waves-light btn"%>
```

Comment:
Refactor notes:
- remove () from commitments_path
- remove ```method: :post``` since button_to is defaulted to post